client/elements/text/sc-simple-text.html

```diff
@@ -323,6 +322,20 @@
               }
           }
 
+          _shouldDisplayBookIcon(divisionId, itemId) {
+              let divisionIdMatch = false;
+              let secondEdition = false;
+              let correctId = false;
+              (divisionId === 'tha' || divisionId === 'thi') ? divisionIdMatch = true : divisionIdMatch = false;
+              itemId.includes('2ed') ? secondEdition = true : secondEdition = false;
+              (itemId.startsWith('pts-vp-pli') || itemId.match(/pts[1-9]/)) ? correctId = true : correctId = false;
+              if (!divisionIdMatch && !secondEdition && correctId) {
```

Comment:
`return (!divisionIdMatch && !secondEdition && correctId)`

Reply:
As we say in Aus: Ta!